packages/logging_cloudwatch/aws_logging_cloudwatch/test/cloudwatch_logger_plugin_test.dart

```diff
@@ -87,11 +158,15 @@ void main() {
           .thenReturn(false);
 
       plugin.enable();
-
       await expectLater(
         plugin.handleLogEntry(errorLog),
         completes,
       );
+      // should not log this because it is below default log level.
```

Comment:
How do you test for that behavior?  It seems the expectLater logic is the same regardless if it should be logged or not?